make_atf.sh

```diff
@@ -27,8 +27,9 @@
 # OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 # SUCH DAMAGE.
 
-tcptestsuite_path=/usr/local/share/tcptestsuite
-packetdrill_path=/usr/local/bin/packetdrill
+: ${PREFIX:=/usr/local}
+tcptestsuite_path=${PREFIX}/share/tcptestsuite
+packetdrill_path=${PREFIX}/bin/packetdrill
```

Comment:
This should be `${LOCALBASE}/bin/packetdrill`.  `LOCALBASE` is the location where other ports are already installed.  `PREFIX` is the location where _this_ port will be installed.

Reply:
Ah, thanks.  Clearly, I work in the ports system just often enough to be dangerous.  :)